### PR TITLE
Support for named export in Node v14.13.0

### PR DESCRIPTION
**Pull request by mcollina**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: examples/simple.mjs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { fastify } from '../fastify.js'
```
Comment (kibertoad) [inline examples/simple.mjs:1]: Should we add comment, explaining that this only works 14.13.0+?

**Review by fox1t — APPROVED** [review]
LGTM

**Comment by fox1t** [comment]
This is huge for the interoperability! I was hoping for this feature to be added to Node.js for months now and I've completely missed the related PR.
🎉

**Comment by fox1t** [comment]
Note: we need to change this "infamous triplet" export all over the place. :)

**Review by RafaelGSS — APPROVED** [review]
LGTM.

**Review by zekth — APPROVED** [review]
LGTM !

**Review by delvedor — APPROVED** [review]
LGTM

**Review by Ethan-Arrowood — APPROVED** [review]
🎉🍻

**Pull request merged**